src/test/java/seedu/address/logic/commands/ClientEditCommandTest.java

```diff
@@ -33,7 +41,12 @@
  */
 public class ClientEditCommandTest {
 
-    private final Model model = new ModelManager(getTypicalTbmManager(), new UserPrefs());
+    private Model model = new ModelManager(getTypicalTbmManager(), new UserPrefs());
```

Comment:
```suggestion
    private Model model;
```

Already done in setUp